pandas/tests/extension/decimal/test_decimal.py

```diff
@@ -115,6 +115,10 @@ def check_reduce(self, s, op_name, skipna):
             expected = getattr(np.asarray(s), op_name)()
             tm.assert_almost_equal(result, expected)
 
+    @pytest.mark.skip("tests not written yet")
+    def check_reduce_and_wrap(self, ser: pd.Series, op_name: str, skipna: bool):
+        pass
```

Comment:
I think it would be good to implement at least one of those, to have coverage of the base class (our own EAs currently (mostly?) override the base class method?) 

(and decimal should be a good candidate, for example just testing that the min/max of DataFrame with decimal column still has decimal dtype)

Reply:
I've added tests for `Decimal`.